reviewing/dependencies-hierarchy/src/getPkgInfo.ts

```diff
@@ -74,7 +74,7 @@ export function getPkgInfo (opts: GetPkgInfoOpts): PackageInfo {
     version = opts.ref
   }
   const fullPackagePath = depPath
-    ? path.join(opts.modulesDir, '.pnpm', depPathToFilename(depPath))
+    ? path.join(opts.virtualStoreDir ?? '.pnpm', depPathToFilename(depPath))
```

Comment:
This doesn't seem correct. It should be probably something like this
```suggestion
    ? path.join(opts.virtualStoreDir ?? path.join(opts.modulesDir, '.pnpm'), depPathToFilename(depPath))
```

Reply:
The `virtualStoreDir` from `readModulesManifest` seems to be an absolute path already, no need to add `modulesDir`

Reply:
Currently run `pd -r --parseable why got`, will output as following:

```
/Documents/test/test-pnpm/test-space
/Documents/test/test-pnpm/test-space/packages/bar
/Documents/test/test-pnpm/test-space/fake_modules/.fake_store/got@2.9.2
/Documents/test/test-pnpm/test-space/packages/bar
/Documents/test/test-pnpm/test-space/fake_modules/.fake_store/got@2.9.2
/Documents/test/test-pnpm/test-space/packages/foo
/Documents/test/test-pnpm/test-space/packages/bar
/Documents/test/test-pnpm/test-space/fake_modules/.fake_store/got@2.9.2
/Documents/test/test-pnpm/test-space/fake_modules/.fake_store/got@3.3.1
```

It doesn't look like show the full path, maybe it should be /Documents/test/test-pnpm/test-space/fake_modules/.fake_store/got@2.9.2/**node_modules/got**

Reply:
add a new commit to show full path(modified a lot of test cases), If it is not suitable, I will revert this commit